iroh-util/src/lock.rs

```diff
@@ -37,21 +40,62 @@ impl ProgramLock {
 
     /// Try to acquire a lock for this program.
     pub fn acquire(&mut self) -> Result<()> {
-        let file = Rc::new(File::create(&self.path)?);
+        let mut file = File::create(&self.path)?;
+        file.write_u32::<LittleEndian>(process::id())?;
```

Comment:
I've considered just writing the PID in plaintext so humans can read it too, which is a double edged sword. My experience in the past with making things like this unreadable as a user is, well, constant irritation. but on the upside it's slightly tougher for users to go poking around with these files in a way that can mess things up.

Reply:
I would recommend just using `.to_string()` (and `parse` for reading). To write human readable pids, this is good for at least us when we debug things :)

Reply:
agreed, change should be in already